app/views/nodes/index.html.slim

```diff
@@ -1,12 +1,65 @@
-h1
-  | Bootstrap cluster
+.alert.alert-info.alert-dismissible role="alert"
+  button.close[type="button" data-dismiss="alert" aria-label="Close"]
+    span[aria-hidden="true"]
+      | &times;
+  i.fa.fa-4x.pull-left aria-hidden="true"
+  span
+    | After choosing the master and clicking "Apply" all other nodes will be set to the minion role
+h1 Nodes
+.row
+  .col-xs-12.discovery-control
+    span #{Minion.count} Nodes found
+    a.btn.btn-default[href="/nodes"] Refresh
+= form_tag(nodes_path, method: "put")
+.nodes-container
+  table.table
+    thead
+      tr
+        th
+          | Id
+        th.text-center
+          | Status
+        th
+          | Hostname
+        th
+          | Role
+        th
+          | Highstate
```

Comment:
I'm not sure what implementation is best in Ruby (single page vs two pages), but in terms of what the user will see, this doesn't match Ken's design for none of the pages.
And as I told you guys already, we're still on time to iterate over the design one more time, but you need to talk with the designer and raise this inconsistencies! He's responsible for the UI of the product, after all.